### PR TITLE
2022.11.0

### PR DESCRIPTION
**Pull request by ELundby45**
# s3fs 2022.11.0

upstream: https://github.com/fsspec/s3fs/tree/2022.11.0
jira: https://anaconda.atlassian.net/browse/PKG-805
`setup.py`: https://github.com/fsspec/s3fs/blob/2022.11.0/setup.py
`requirements.txt`: https://github.com/fsspec/s3fs/blob/2022.11.0/requirements.txt
changelog: https://github.com/fsspec/s3fs/compare/2022.10.0...2022.11.0

## Changes
- Update version and SHA
- Minor whitespace and about section updates
- Adjust `aiobotocore` pinning to match upstream

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
 
   run:
     - python
-    - aiobotocore ==2.4.0
+    - aiobotocore ~=2.4.0
```
Comment (cbouss) [inline recipe/meta.yaml]: I don't think conda understands ~= (https://peps.python.org/pep-0440/#compatible-release)
Instead you could set: `- aiobotocore >=2.4.0,==2.4.*`

Comment (ELundby45) [reply recipe/meta.yaml]: Good to know! Thank you.

**Review by cbouss — APPROVED** [review]
LGTM!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ requirements:
 
   run:
     - python
-    - aiobotocore ==2.4.0
+    - aiobotocore >=2.4.0,==2.4.*
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The correct pinning will be `aiobotocore >=2.4.0,<2.4.1` or `aiobotocore ==2.4.*` or `aiobotocore 2.4.*`

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml]: yep

**Pull request merged**